### ISSUE TITLE
Proposal: Minimap

### ISSUE THREAD

**Issue by kd7uiy** [TeamPorcupine/ProjectPorcupine#566]
It would be nice to have a minimap to show how the whole layout looks. I talk a bit about this in a blog entry at http://www.completeunitydeveloper.com/blog/mini-map-and-custom-texture-creation-in-unity on how to do it, there are other ways as well. This is probably a more advanced change than some, so I suggest we think if there are any gotchas.

There is another way that could be done, which would be to use a render textured camera set at such a level where a small image of the entire map can be seen. I suspect we would get better performance using a custom texture, but I'm not really sure.

I have no intention of doing this anytime in the next week, if it's not done by then then I will likely consider it.

**Comment by spexiono** [comment]
Good idea. Perhaps to improve it's functionality adding filters (In the minimap) for units (Blue friendly, Red enemy etc) and/or areas (like how Rimworld has a "home" zone). Probably Others if people come up with useful ideas.

**Comment by kd7uiy** [comment]
More important that the exact colors for now is how those colors will be determined. We could simply have 1 pixel per tile, and use the color of the middle of the sprite, or maybe the mean of the sprite? Probably should be a few pixels at least. Hmmm...

**Comment by Mizipzor** [comment]
This is not easy to do, add it as a feature on that vote-on-feature page. :)

**Comment by UnknownSoldier1** [comment]
@kd7uiy I would say determine the pixel color by the average color of the tile sprite.
